src/components/MessageInput/hooks/useUserTrigger.ts

```diff
@@ -126,7 +126,7 @@ export const useUserTrigger = <
 
       const filterMutes = (data: UserResponse<Us>[]) => {
         if (!mutes.length) return data;
-        return data.filter((suggestion) => mutes.some((mute) => mute.target.id === suggestion.id));
+        return data.filter((suggestion) => mutes.some((mute) => mute.target.id !== suggestion.id));
```

Comment:
I don't think this is quite right either. Shouldn't the `some` here be `every` instead? Otherwise if I have two users with different `id`s muted, this filter will always return true.

```js
return data.filter((suggestion) => mutes.some((mute) => mute.target.id !== suggestion.id));
```

Thanks for your help on this!

Reply:
You're right - just added a new PR. Thanks!